blackbox/docs/sql/partitioned_tables.txt

```diff
@@ -438,6 +438,21 @@ done by using the ``ONLY`` keywword.
     cr> ALTER TABLE ONLY parted_table SET (number_of_replicas = 1);
     ALTER OK, -1 rows affected (... sec)
 
+Closing and Opening a Partition
+-------------------------------
+
+A single partition within a partitioned table can be opened and closed in the
+same way a normal table can.
+
+::
+
+    cr> ALTER TABLE parted_table PARTITION (day=1396915200000) CLOSE;
+    ALTER OK, -1 rows affected (... sec)
+
+This will prevent inserting into this particular partition, as well as removing
+the partition's data from ``SELECT`` operations conducted on the main partitioned
+table.
```

Comment:
again, all will fail on this partition (partition is not included in any query on the partitioned table)